$OpenBSD: patch-configure,v 1.29 2016/06/14 14:11:08 dcoppa Exp $

- Don't reset CPPFLAGS
- XXX: boost/intrusive/detail/array_initializer.hpp:62:7: error: exception handling disabled, use -fexceptions to enable

--- configure.orig	Mon May  2 22:05:37 2016
+++ configure	Tue Jun 14 15:58:02 2016
@@ -12472,7 +12472,7 @@ if test x$enable_aac = xyes; then
 	oldcppflags=$CPPFLAGS
 	CFLAGS="$CFLAGS $FAAD_CFLAGS"
 	LIBS="$LIBS $FAAD_LIBS"
-	CPPFLAGS=$CFLAGS
+	CPPFLAGS="$CPPFLAGS $CFLAGS"
 	ac_fn_c_check_header_mongrel "$LINENO" "faad.h" "ac_cv_header_faad_h" "$ac_includes_default"
 if test "x$ac_cv_header_faad_h" = xyes; then :
 
@@ -17130,6 +17130,7 @@ fi
 done
 
 
+if false; then
 if test x$no_exceptions = xyes; then
 
 
@@ -17195,7 +17196,7 @@ else
 fi
 
 done
-
+fi
 
 
 
